2.7/README.md

```diff
@@ -43,7 +43,7 @@ in Openshift can be achieved with the following step:
 The same application can also be built using the standalone [S2I](https://github.com/openshift/source-to-image) application on systems that have it available:
 
     ```
-    $ s2i build https://github.com/sclorg/s2i-python-container.git --context-dir=2.7/test/setup-test-app/ rhscl/python-27-rhel7 python-sample-app
+    $ s2i build https://github.com/sclorg/s2i-python-container.git --context-dir=2.7/test/setup-test-app/ <image_name> python-sample-app
```

Comment:
I don't see any advantage of this cleanup. Copy & paste of this command was easy. in new format you have to overwrite <image_name>

Reply:
Yes, but in many cases, the example is not valid. For example, we have Python 3.8 still supported only on Fedora but the example is the same containing `rhscl/python-38-rhel7`. The same applies on Python 3.7 which won't be available in RHEL at all. I provided a set of links where to download images for Centos, Fedora or RHEL and I can add a note about `<image_name>`, see the discussion above.